### PR TITLE
fixed ambiguity in documentation with `<composer_root>` in name modulite section

### PR DESCRIPTION
**Pull request by Hidanio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,6 +231,7 @@ It's done in the following way. At a root folder, near `composer.json`, place a
 * **force-internal** also works
 * **require** leave blank, it's filled from "requires" of *composer.json*
 
+**Important**: You should write **name** = "<composer_root>" as is. You should not replace <composer_root> with your package name from composer.
```
Comment (Danil42Russia) [inline README.md]: `"<composer_root>"`

File: docs/index.html [hunk 2]
```diff
@@ -948,6 +948,8 @@ <h2 class="title">export из Composer-пакета</h2>
             <li><b>force-internal</b> тоже работает</li>
             <li><b>require</b> оставляем пустым, он всё равно берётся из composer.json</li>
         </ul>
+        <p><b>Важно</b>: вам следует писать <b>name</b> = "&lt;composer_root&gt;" как есть. Не следует заменять <composer_root> именем вашего пакета из композера.
```
Comment (Danil42Russia) [inline docs/index.html]: replace `<composer_root> ` with `"&lt;composer_root&gt;"`

File: README.md (hunk 1 above)
Comment (Hidanio) [reply README.md]: Done

File: docs/index.html (hunk 2 above)
Comment (Hidanio) [reply docs/index.html]: Done

**Review by Danil42Russia — APPROVED** [review]

**Pull request merged**